docs/ref/utils.txt

```diff
@@ -729,6 +729,15 @@ escaping HTML.
 
     Outputs a string in the format ``Wdy, DD Mon YYYY HH:MM:SS GMT``.
 
+.. function:: content_disposition_header(as_attachment, filename)
+
+    .. versionadded:: 4.2
+
+    Constructs a ``Content-Disposition`` HTTP header value to match the
+    :rfc:`6266` specification.  Returns ``None`` if ``as_attachment`` is
+    ``False`` and ``filename`` is ``None``, otherwise returns a string suitable
+    for storing in the ``Content-Disposition`` header.
```

Comment:
What do other think about putting this in the docstring as well?

Reply:
I pushed small edits to docs and `content_disposition_header()`'s docstring.